testutil/singlevm/setup.sh

```diff
@@ -375,9 +458,17 @@ qemu-img convert -f raw -O qcow2 "$ciao_cnci_image" "$ciao_cnci_image".qcow
 
 #Clear
 cd "$ciao_bin"
-LATEST=$(curl https://download.clearlinux.org/latest)
 
-if [ $download -eq 1 ] || [ ! -f clear-"${LATEST}"-cloud.img ] 
+if [ $download -eq 1 ]
+then
+	LATEST=$(curl https://download.clearlinux.org/latest)
+else
+	# replace this will a function that looks in ~local cache
+	# for the last version of clear to use.
+	LATEST="12620"
+fi
+
+if [ $download -eq 1 ] && [ ! -f clear-"${LATEST}"-cloud.img ]
```

Comment:
This will only download now if downloads are forced on which is not the default.